src/Altinn.Common.PEP/Altinn.Common.PEP/Authorization/ResourceAccessHandler.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Helpers;
+using Altinn.Common.PEP.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Altinn.Common.PEP.Authorization
+{
+    /// <summary>
+    /// AuthorizationHandler that is created for handling access to app.
```

Comment:
Documentation of class and initializer could use an update (AppAccessHandler ref)

Reply:
Fixed